.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and test
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:foxy
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Clone dependency packages
+        run: |
+          mkdir dependency_ws
+          vcs import dependency_ws < build_depends.repos
+
+      - name: Install missing dependencies
+        run: |
+          sudo apt-get update
+          rosdep update
+          DEBIAN_FRONTEND=noninteractive rosdep install --from-paths obstacle_stop_planner_refine --ignore-src --rosdistro foxy -y --skip-keys=vehicle_info_util --skip-keys=autoware_planning_msgs --skip-keys=autoware_perception_msgs --skip-keys=autoware_debug_msgs
+
+      - name: Build
+        run: |
+          . /opt/ros/foxy/setup.sh
+          colcon build \
+            --catkin-skip-building-tests \
+            --packages-up-to obstacle_stop_planner_refine \
+            --event-handlers console_cohesion+
```

Comment:
IMO: It would be better to specify:
1. `--cmake-args -DCMAKE_BUILD_TYPE=Release` or
2. `--cmake-args -DCMAKE_BUILD_TYPE=Debug` or
3. `--cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo`

It depends on the purpose of this CI. If just for a test, item 2 would be better.

Reply:
@kmiya Thanks. I use # 2 at 3b8ee25.